README.md

```diff
@@ -325,20 +323,46 @@ snakemake \
-[Oligomap][oligomap], are used. All the resulting alignments are merged such 
-that only the best alignments of each read are kept (smallest edit distance).
-Alignments are intersected with the user-provided, pre-processed miRNA
-annotation file using [BEDTools][bedtools]. Counts are tabulated separately for
-reads consistent with either miRNA precursors, mature miRNA and/or isomiRs.
-Finally, ASCII-style alignment pileups are optionally generated for
-user-defined regions of interest.
-
-> **NOTE:** For a detailed description of each rule, please, refer to the
-> [workflow documentation](pipeline_documentation.md)
+The _MIRFLOWZ_ workflow initially processes and indexes the genome resources
+provided by the user. The regions corresponding to mature miRNAs are extended
+on both sides to accommodate isomiR species with shifted start and/or end
+positions. If necessary, pri-miR loci are similarly extended to adjust to the
+new miRNA coordinates.
+
+Subsequently, the user-provided short-read small RNA-seq libraries undergo
+quality filtering if a FASTQ file is provided. Alternatively, adapters are
+directly removed. The resulting reads are independently mapped to both the
+genome and the transcriptome using two distinct aligners: [Segemehl][segemehl]
+and our in-house tool [Oligomap][oligomap]. After the mapping, only the best
+alignments for each read, determined by the smallest edit distance, are
+retained by merging and filtering the resulting alignments into a single file.
+
+The collection of resulting alignments is then reduced to contain only unique
+entries. Due to the short length of the reads and the sequence similarity among
+miRNAs, the number of alignments can be high. Therefore, reads aligned beyond a
+specified threshold are discarded. To address multimapping, alignments with the
+fewest indels are preserved. These alignments are subsequently intersected with
+the user-provided, pre-processed miRNA annotation files using
+[BEDTools][bedtools]. Note that an alignment will not contribute to the final
+count if its start and/or end positions differ significantly from the provided
+miRNA annotations, beyond the extension applied to the mature miRNA start
+and/or end positions, or by 1 if no extension was applied. Conversely, a
+retained read contributes 1/n to all the annotated miRNA species it aligns
+with, where `n` is the number of genomic and/or transcriptomic loci it aligns
+to.
+
+_MIRFLOWZ_ employs an unambiguous notation to classify isomiRs using the format
+`miRNA_name|5p-shift|3p-shift|CIGAR|MD`, where `5p-shift` and `3p-shift`
+represent the differences between the annotated mature miRNA start and end
+positions and those of the alignment, respectively.
+
+Counts are tabulated separately for reads consistent with either
+miRNA precursors, mature miRNA and/or isomiRs and all library counts are
+fused into a single table. Finally, ASCII-style alignment pileups are
+optionally generated for user-defined regions of interest.
```

Comment:
"Finally, to visualize the distribution of read alignments around miRNA loci, ASCII-style alignment pileups are [...]"